packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -23,10 +41,40 @@ export default function ErrorComponentPreview(props: {
   }, []);
 
   const rect = containerRef.current?.getBoundingClientRect();
-  const tooltipPosition = rect && {
-    x: (rect.right + rect.left) / 2,
-    y: rect.top - 40,
-  };
+  const tooltipPosition = useMemo(() => {
```

Comment:
it looks like tooltipPosition is only used when setting the tooltip props. can we move this calculation into the useEffect?